storybookjs/storybook
# UI: Fix document title not updating

Issue: #866 

## What I did
I extracted storyInfo and passed it to `<Helmet />` so that it can refresh document title.

## How to test

- Is this testable with Jest or Chromatic screenshots? Should later add tests to `lib/ui` package
- Does this need a new example in the kitchen sink apps? No
- Does this need an update to the documentation? No
